frontend/store/authorizations.ts

```diff
@@ -243,4 +243,10 @@ export const actions = {
             commit('setIsAllowanceAmountLoading', false);
         }
     },
+    async setup({ commit, dispatch }: ActionContext<State, State>) {
+        commit('reset');
+
+        await dispatch('refetch');
+        await dispatch('fetchAllowanceAmount');
```

Comment:
I believe you also need to fetch authorizations for wallet and specific opened collateral. This logic is already implemented for "wallet switch" so you would need to refactor it into a single place

Reply:
Yes, seems like it. Do you want to move `fetchAllowanceAmount` call to the `refetch` as well?

Reply:
Sure, I think that makes sense